lib/GenomeFileUtil/core/GenbankToGenome.py

```diff
@@ -113,52 +114,16 @@ def __init__(self, config):
 
     def import_genbank(self, params):
         print('validating parameters')
-        mass_params = self._set_up_single_params(params)
+        mass_params = set_up_single_params(
+            params, _WSNAME, self._validate_params, self.dfu.ws_name_to_id
+        )
         return self._import_genbank_mass(mass_params)[0]
 
     def import_genbank_mass(self, params):
```

Comment:
I don't remember the state of the tests for this function and what tests still need to be added. Can you remind me?

Reply:
Which function are you referring to?

Reply:
import_genbank_mass

Reply:
The focus of this PR is batch saving genomes. The import_genbank_mass PR has been merged: [link](https://github.com/kbaseapps/GenomeFileUtil/pull/208). I also wrote many many tests...

Reply:
Great. What tests still need to be added though?

Reply:
https://github.com/kbaseapps/GenomeFileUtil/pull/212#discussion_r1767301229

Reply:
That doesn't answer the question

Reply:
No tests need to be added for import_genbank_mass.

However, we need to add rigorous tests for check_save_one_genome_output. I'm not sure if we want to do this in this iteration or document it for the future.

Reply:
Document only

Reply:
Github issue: https://github.com/kbaseapps/GenomeFileUtil/issues/214
JIRA ticket: https://kbase-jira.atlassian.net/jira/software/projects/SECURITY/boards/87?selectedIssue=SECURITY-25

Reply:
:+1: